dcap_ql/sys/types/build.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Builds the FFI type bindings for the dcap quote library of the Intel SGX SDK
+
+use bindgen::callbacks::ParseCallbacks;
+
+const DCAP_QL_TYPES: &[&str] = &[
```

Comment:
This seems to be missing `sgx_target_info_t` (an argument type for `sgx_qe_get_target_info`), `_sgx_ql_pck_cert_id_t` (defined in `sgx_ql_lib_common.h`), `sgx_ql_logging_callback_t` (defined in `sgx_ql_lib_common.h`), and a number of types from `sgx_pce.h`.

It's a bit unclear to me whether `sgx_pce.h` should be included transitively, so that last bit might be irrelevant.

Reply:
In light of the below comment I should say that all of these statements are based on the dcap_1.11_reproducible tag checked out by the linux-sgx repo.

Reply:
`sgx_target_info_t` comes from the `mc-sgx-core-sys-types`
I've added the `sgx_pce.h` types since one of them is used directly.

I think we'll need a follow up to update all the readme's for the version of SGX/DCAP being supported.